Rollbar/NSJSONSerialization+Rollbar.h

```diff
@@ -12,9 +12,15 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-+ (nullable NSData *)dataWithJSONObject:(id)obj options:(NSJSONWritingOptions)opt error:(NSError **)error safe:(BOOL)safe;
++ (nullable NSData *)dataWithJSONObject:(id)obj
```

Comment:
Is your editor doing this formatting automatically?

Reply:
somewhat, I had to hit enter key between the arguments. do you prefer to keep original formatting?